browser/brave_ads/android/java/org/chromium/chrome/browser/dialogs/BraveAdsNotificationDialog.java

```diff
@@ -70,44 +81,73 @@ public static void showAdNotification(Context context, final String notification
         window.setFlags(WindowManager.LayoutParams.FLAG_NOT_TOUCH_MODAL,
                 WindowManager.LayoutParams.FLAG_NOT_TOUCH_MODAL);
 
-        window.clearFlags(WindowManager.LayoutParams.FLAG_DIM_BEHI‌​ND);
-
-        ImageView closeButton = mAdsDialog.findViewById(R.id.brave_ads_custom_notification_close_button);
+        window.clearFlags(WindowManager.LayoutParams.FLAG_DIM_BEHIND);
+        window.setBackgroundDrawable(new ColorDrawable(Color.TRANSPARENT));
 
         ((TextView) mAdsDialog.findViewById(R.id.brave_ads_custom_notification_header)).setText(title);
         ((TextView) mAdsDialog.findViewById(R.id.brave_ads_custom_notification_body)).setText(body);
 
-        closeButton.setOnClickListener(new View.OnClickListener() {
+        mNotificationId = notificationId;
+        mAdsDialog.findViewById(R.id.brave_ads_custom_notification_popup).setOnTouchListener(new View.OnTouchListener() {
             @Override
-            public void onClick(View view) {
-                mAdsDialog.dismiss();
-                mAdsDialog = null;
-                BraveAdsNativeHelper.nativeOnCloseAdNotification(
-                        Profile.getLastUsedRegularProfile(), mNotificationId, true);
-                mNotificationId = null;
+            public boolean onTouch(View v, MotionEvent event) {
```

Comment:
Are we handling swipes up and down? Usually notifications are dismissed by swiping left and right on Android.

Reply:
Ads team wanted to copy the behavior from iOS. @tmancey would you chime in? I don't know if I can make it in time for trains, but we can let it sit and if users want left/right swipe I can do so.

Reply:
@jenn-rhim Your thoughts? For me I like to follow the systems default, which as above is left/right